.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "eslint.enable": true,
+  "eslint.autoFixOnSave": true,
```

Comment:
do we need vs code plugin for this?

Reply:
You don't "need" the vscode plugin, but it helps.